master/internal/api_generic_tasks.go

```diff
@@ -231,9 +232,6 @@ func (a *apiServer) CreateGenericTask(
 		}
 
 		forkedConfig = []byte(resp.Config)
-		if err != nil {
-			return nil, err
-		}
```

Comment:
Why has this disappeared?

Reply:
look earlier in this code block, and you'll see that it already checked the error & returned there -- so this code block is essentially a `nil != nil` check, which can never be true, and thus not needed code